index.html

```diff
@@ -1,88 +1,89 @@
-<!DOCTYPE html
-	PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-<html lang="en" dir="ltr" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
 
-<head>
-	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <title>WikiTree: View registry</title>
 
-	<title>WikiTree Dynamic Family Tree</title>
+    <link rel="icon" href="https://www.wikitree.com/favicon.ico" />
 
-	<base target="_parent">
-	<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
-	<script src="TreeAPI.js"></script>
-	<script src="Tree.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/d3/3.5.5/d3.min.js"></script>
```

Comment:
Script tags should have a mandatory `type` attribute. eg `type="application/javascript"`

Reply:
Not really true:

> Omitted or a JavaScript MIME type: This indicates the script is JavaScript. The HTML specification urges authors to omit the attribute rather than provide a redundant MIME type. 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script#attr-type